### DISCUSSION TITLE
Add support for the webm media file format

### DISCUSSION THREAD

**Discussion by Raphexion** [community/community#8864] [Pull Requests]
I really love the feature that it is possible to attach images and videos to pull-requests and issues. However, I've noticed that the [webm](https://en.wikipedia.org/wiki/WebM) format isn't supported.

Would it be possible for Github to add support for the webm format?

**Comment by TheRadicalDreamer** [comment]
It is really silly, but it DOES support WEBM. If you rename your file to .mp4, you can upload it, and it will be properly played on the comments!

Come on @github-staff please do not filter WEBM when uploading it to comments

**Comment by qcasey** [comment]
I think WEBM support might be especially important now that Gnome 42 is rolling out broadly, in which the default screen recorder saves to `.webm`.

Renaming `.webm` to `.mp4` seems to break playback in Safari, but does work in FF/Chromium.

**Comment by Drew-Daniels** [comment]
Bump

**Comment by joan38** [comment]
+1 Gifs are so inefficient

**Comment by TheRadicalDreamer** [comment]
Chromeos have its screenrecorder in WEBM, so it is already a standard in many devices

**Reply by mloskot** [reply]
Ubuntu default screencast produces WEBM too.

**Comment by phillip-kruger** [comment]
+1

**Comment by codingjoe** [comment]
And AVIF, it's 2025… AVIF is Baseline 2024.
